src/asInput/index.jsx

```diff
@@ -125,6 +125,20 @@ const asInput = (WrappedComponent, labelFirst = true) => {
       return desc;
     }
 
+    getLabel() {
+      return (
+        <label
+          id={`label-${this.state.id}`}
+          htmlFor={this.state.id}
+          className={[classNames({
+            [styles['form-check-label']]: inputType === 'checkbox',
+          })]}
+        >
+          {this.props.label}
+        </label>
+      );
+    }
+
```

Comment:
Moved logic for creating the label element out to help with readability and styling. As noted in the above comment, this will likely need a different class name / formatting for the upcoming Search input type that Dahlia will be working on shortly.